extensions/portuguese-primeira-liga/package.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "portuguese-primeira-liga",
+  "title": "Portuguese Primeira Liga",
+  "description": "View the latest Portuguese Primeira Liga standings.",
+  "icon": "icon.png",
+  "author": "danielsequeira",
+  "owner": "raycast",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "standings",
+      "title": "Standings",
+      "subtitle": "Portuguese Primeira Liga",
+      "description": "View the latest Portuguese Primeira Liga standings.",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "apiKey",
+      "type": "password",
+      "required": true,
+      "title": "API Key",
+      "description": "Your Football Data API Key",
+      "link": "https://www.football-data.org/",
```

```suggestion
      "description": "Your football data API key (Get one from: https://www.football-data.org/)",
```
There is no [link property](https://developers.raycast.com/information/manifest#preference-properties)